modules/ossm-config-openshift-monitoring-only.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/metrics/ossm-metrics-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-only_{context}"]
+= Configuring OpenShift Monitoring with Service Mesh
+
+// This is WITHOUT Kiali.
+// TP1 content. TP banner is persistent and handled by Tim O'Keefe in a different PR.
+// Possible file name may change
+// Possible assembly file may change
+// Assemblies, topic map info needs to be worked out still for 3.0.
+
+The following steps show how to integrate {SMProductName} with user-workload monitoring.
+
+.Prerequisites
+
+* {SMProductName} is installed.
+
+* User-workload monitoring is enabled. See link:https://docs.openshift.com/container-platform/4.16/observability/monitoring/enabling-monitoring-for-user-defined-projects.html[Enabling monitoring for user-defined projects].
+
+.Procedure
+
+. Apply a `ServiceMonitor` object to monitor the Istio control plane:
++
+.Example `ServiceMonitor` object
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: istiod-monitor
+  namespace: istio-system
+spec:
+  targetLabels:
+  - app
+  selector:
+    matchLabels:
+      istio: pilot
+  endpoints:
+  - port: http-monitoring
+    interval: 30s
```

Comment:
add "# ..." as this might not be a complete YAML file? 

See [doc_guidelines.adoc](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#signs-and-symbols-in-code-blocks)

Reply:
This is a complete yaml file. It's just short :)